index.html

```diff
@@ -1,19 +1,71 @@
-<!doctype html>
-
-<html lang="en">
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
-
+   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+ 
 </head>
-
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
+ 
+ <body>
+      <!-- Copy and paste your HTML from the first UI project here -->
+      <header>
+         <nav class="nav">
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+         </nav>
+         <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
*Small detail but consider adjusting your logo image. It looks good on mobile, but it stretches vertically in desktop view and not centered in tablet.